app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  has_one :passenger
+  has_many :invoices
+  has_many :itineraries, through: :invoices
+
+  validates :email, presence: true, uniqueness: true
+
+  def flights
+    passenger.flights
+  end
```

Comment:
it looks like this could be replaced with
```suggestion
  has_many :flights, through: :passenger
```

Reply:
Done!